ruff.toml

```
@@ -391,6 +391,9 @@ extend-ignore-names = ["HWR"] # To avoid "N814": Camelcase `HardwareRepository`
     "E501",
     "UP032",
 ]
+".github/scripts/*.py" = [
+    "INP001"
```

Add a trailing comma:

```suggestion
    "INP001",
```